### PR TITLE
ZH-trans: add /kubeadm/generated/...

### PR DESCRIPTION
**Pull request by xichengliudui**
zh-trans: add kubeadm/generated/...
[File directory](https://github.com/kubernetes/website/tree/master/content/en/docs/reference/setup-tools/kubeadm/generated)

### CODE REVIEW

**Comment by chenrui333** [comment]
@xichengliudui for chinese content, you dont have to assign outside Chinese i18n team, you can cross post PR review request into wechat group discussions if needed, thanks!

**Comment by chenrui333** [comment]
I might just have to suggest that we probably dont need to translate `--attribute string` to something like `--attribute 字符串` due to the type explicitness

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">一组键=值对，用于描述各种特性的特性门。选项有：<br/>Auditing=true|false (ALPHA - default=false)<br/>CoreDNS=true|false (default=true)<br/>DynamicKubeletConfig=true|false (BETA - default=false)</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">列表帮助</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md]: list 操作的帮助信息

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_pull.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+
+拉出 kubeadm 使用的图像。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_pull.md]: pull -> 拉取

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_pull.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">一组键=值对，用于描述各种特性的特性门。选项：<br/>Auditing=true|false (ALPHA - default=false)<br/>CoreDNS=true|false (default=true)<br/>DynamicKubeletConfig=true|false (BETA - default=false)</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">拉取镜像帮助</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_pull.md]: pull 操作的帮助信息

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+<!--
+This command lets you convert configuration objects of older versions to the latest supported version,
+locally in the CLI tool without ever touching anything in the cluster.
+In this version of kubeadm, the following API versions are supported:
+-->
+
+- kubeadm.k8s.io/v1alpha2
+- kubeadm.k8s.io/v1alpha3
+
+此外，kubeadm 只能写出版本"kubeadm.k8s.io/v1alpha3"的配置，但读取这两种类型，不管你传递给
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]:
```suggestion
此外，kubeadm 只能写出版本"kubeadm.k8s.io/v1alpha3"的配置，但能够读取这两种类型，不管你传递给
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+——old-config 的参数是什么版本，在写入 stdout 或 --new-config 如果指定时，API 对象都是
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]:
```suggestion
--old-config 的参数是什么版本，在写入 stdout 或 --new-config（如果指定），API 对象都会被
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+读取、反序列化、默认设置、转换、验证并在写入 stdout 时重新序列化。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]:
```suggestion
读取、反序列化、应用默认设置、执行版本转换与合法性验证，并在输出时重新序列化。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+<!--
+Further, kubeadm can only write out config of version "kubeadm.k8s.io/v1alpha3", but read both types.
+So regardless of what version you pass to the --old-config parameter here, the API object will be
+read, deserialized, defaulted, converted, validated, and re-serialized when written to stdout or
+--new-config if specified.
+-->
+
+换句话说，这个命令的输出就是 kubeadm 在内部读取的内容
+提交这个文件到 "kubeadm init"
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]: 标点符号？

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">迁移帮助</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]: migrate 操作的帮助信息

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+    </tr>
+
+    <tr>
+      <td colspan="2">--new-config string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">使用新的 API 版本生成的 kubeadm 配置文件的路径是可选的。如果没有指定输出路径将被输出到标准路径。</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]:
```suggestion
      <td></td><td style="line-height: 130%; word-wrap: break-word;">使用新的 API 版本生成的 kubeadm 配置文件的路径。这个路径是可选的。如果没有指定，输出将被写到 stdout。</td>
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+    </tr>
+
+    <tr>
+      <td colspan="2">--old-config string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">使用旧 API 版本的 kubeadm 配置文件的路径应该进行转换。flag 参数是强制性的。</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_migrate.md]:
```suggestion
      <td></td><td style="line-height: 130%; word-wrap: break-word;">使用旧 API 版本的、需要进行转换的 kubeadm 配置文件路径。此参数是必需的。</td>
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+-->
+
+### 概要
+
+<!--
+### Synopsis
+-->
+
+
+此命令打印用于'kubeadm init'和'kubeadm upgrade'的默认 InitConfiguration 对象，以及用于 'kubeadm join' 的默认 JoinConfiguration 对象。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md]:
```suggestion
此命令打印用于 'kubeadm init' 和 'kubeadm upgrade' 的默认 InitConfiguration 对象，以及用于 'kubeadm join'  的默认 JoinConfiguration 对象。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+<!--
+This command prints the default InitConfiguration object that is used for 'kubeadm init' and 'kubeadm upgrade',
+and the default JoinConfiguration object that is used for 'kubeadm join'.
+-->
+
+注意，敏感值如引导令牌字段被替换为简单的值如{"abcdef.0123456789abcdef" "" "nil" <nil> [] []}以通过验证但是
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md]:
```suggestion
注意，如引导令牌字段这类敏感值会被替换为简单的值，如{"abcdef.0123456789abcdef" "" "nil" <nil> [] []}，目的是在通过合法性验证的同时避免执行创建令牌这类真实计算。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--api-objects stringSlice</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">API 对象打印默认值的逗号分割列表。可用值：[InitConfiguration ClusterConfiguration JoinConfiguration KubeProxyConfiguration KubeletConfiguration MasterConfiguration]。没有设置 flag 参数表示'打印所有已知对象'</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md]: 分割-> 分隔

Comment (tengqm) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md]: > 没有设置 flag 参数表示'打印所有已知对象'

此参数未设置时表示“打印所有已知对象”

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+    </tr>
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">打印默认值帮助</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_print-default.md]: print-default 操作的帮助信息

**Comment by xichengliudui** [comment]
@tengqm ,修改完毕，感谢提出的意见。

**Pull request merged**